atlite/gis.py

```diff
@@ -121,67 +122,18 @@ def compute_indicatormatrix(orig, dest, orig_proj='latlong', dest_proj='latlong'
 
     dest = reproject_shapes(dest, dest_proj, orig_proj)
     indicator = sp.sparse.lil_matrix((len(dest), len(orig)), dtype=np.float)
+    tree = STRtree(orig)
+    idx = dict((id(o), i) for i, o in enumerate(orig))
 
-    try:
-        from rtree.index import Index
-
-        idx = Index()
-        for j, o in enumerate(orig):
-            idx.insert(j, o.bounds)
-
-        for i, d in enumerate(dest):
-            for j in idx.intersection(d.bounds):
-                o = orig[j]
+    for i, d in enumerate(dest):
+        for o in tree.query(d):
+            if o.intersects(d):
```

Comment:
Did you test whether the inclusion of this `if` branch speeds things up?

Because with rtree, it was actually slower to test the intersection first and the result does not change since if they dont intersect the intersection area is 0 and `lil_matrix` suppresses zero values.

Reply:
yes it speeds up, about 25 %. 

Here the examples using 256 shapes from pypsa-eur clustering

With `if` 
```python 
%time indicator_matrix = cutout.indicatormatrix(shapes.geometry)
CPU times: user 3.53 s, sys: 3.84 ms, total: 3.54 s
Wall time: 3.53 s
```

Without `if`
```python
%time indicator_matrix = cutout.indicatormatrix(shapes.geometry)
CPU times: user 4.81 s, sys: 4.7 ms, total: 4.82 s
Wall time: 4.81 s
```